database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
+  s_subscription_object jsonb NOT NULL,
+  CHECK (s_subscription_object->>'id' = s_id)
```

Comment:
I think @v-stepanov really wants to use only the subscription IDs, so in this case it would be a UUID (what I can see from the code)